src/braket/ahs/discretization_types.py

```diff
@@ -0,0 +1,14 @@
+from dataclasses import dataclass
+from typing import Any
+
+
+class DiscretizationError(Exception):
+    """Raised if the discretization of the numerical values of the AHS program fails."""
+
+    pass
+
+
+@dataclass
+class DiscretizationProperties:
```

Comment:
Docstrings?

Reply:
Please check.

Reply:
Could we please include some descriptions for lattice and rydberg. They are marked with type Any and carry no information on what they are supposed to contain/represent.